opendevin/sandbox/docker/ssh_box.py

```diff
@@ -411,7 +411,12 @@ def close(self):
         "Interactive Docker container started. Type 'exit' or use Ctrl+C to exit.")
 
     # Initialize required plugins
-    ssh_box.init_plugins([JupyterRequirement()])
+    ssh_box.init_plugins([JupyterRequirement(), SWEAgentCommandsRequirement()])
+    logger.info(
+        '--- SWE-AGENT COMMAND DOCUMENTATION ---\n'
+        f'{SWEAgentCommandsRequirement().documentation}\n'
+        '---'
+    )
```

Comment:
Eventually we should consider pulling this `main` logic into sandbox.py, and making both sandbox type and the plugin list a command-line arg!